Signal/src/textsecure/Storage/TSStorageManager+messageIDs.m

```diff
@@ -0,0 +1,36 @@
+//
+//  TSStorageManager+messageIDs.m
+//  Signal
+//
+//  Created by Frederic Jacobs on 24/01/15.
+//  Copyright (c) 2015 Open Whisper Systems. All rights reserved.
+//
+
+#import "TSStorageManager+messageIDs.h"
+
+#define TSStorageParametersCollection @"TSStorageParametersCollection"
+#define TSMessagesLatestId            @"TSMessagesLatestId"
+
+@implementation TSStorageManager (messageIDs)
+
++ (NSString*)latestTimestampWithTransaction:(YapDatabaseReadWriteTransaction*)transaction {
```

Comment:
OK I get why you called this function previously now, however it should be refactored to be renamed to something like latestMessageIdWithTransaction